SwrveConversationSDK/src/main/java/com/swrve/sdk/SwrveConversationConstants.java

```diff
@@ -0,0 +1,9 @@
+package com.swrve.sdk;
+
+/**
+ * Used internally
+ */
+public class SwrveConversationConstants {
+    public static final String EXTRA_CONVERSATION_KEY = "conversation";
+    public static final String EXTRA_CONVERSATION_VERSION_KEY = "conversation_version";
```

Comment:
Its used as a key to put SwrveConversation serialized object into an extra in the intent to start conversationactivity. Used by unity also. So best to use a constant.